corral/vcs/git.pony

```diff
@@ -3,6 +3,7 @@ use "../regex"
 use "../util"
 
 class val GitVcs is Vcs
+  """Git implementation of Vcs"""
```

Comment:
I think this should be 3 lines but the style guide isn't clear. @Theodus we should clarify that.

style guide is also unclear about if a newline after the """" matters. there's a mix in this project.